homeassistant/components/nextcloud/coordinator.py

```diff
@@ -19,18 +19,17 @@ class NextcloudDataUpdateCoordinator(DataUpdateCoordinator[dict[str, Any]]):
     """Nextcloud data update coordinator."""
 
     def __init__(
-        self, hass: HomeAssistant, ncm: NextcloudMonitor, config: ConfigType
+        self, hass: HomeAssistant, ncm: NextcloudMonitor, entry: ConfigEntry
     ) -> None:
         """Initialize the Nextcloud coordinator."""
-        self.config = config
         self.ncm = ncm
-        self.url = config[CONF_URL]
+        self.url = entry.data[CONF_URL]
```

Comment:
codecov/patch complained here. Maybe `coordinator` and `entity` should have been added to coverage ignore list in the previous refactoring PRs.